crates/kitsune_p2p/types/src/config.rs

```diff
@@ -83,17 +83,17 @@ pub mod tuning_params_struct {
         /// Default agent count for remote notify. [Default: 5]
         default_notify_remote_agent_count: u32 = 5,
 
-        /// Default timeout for remote notify. [Default: 1000ms]
-        default_notify_timeout_ms: u32 = 1000,
+        /// Default timeout for remote notify. [Default: 30s]
+        default_notify_timeout_ms: u32 = 1000 * 30,
 
-        /// Default timeout for rpc single. [Default: 2000]
-        default_rpc_single_timeout_ms: u32 = 2000,
+        /// Default timeout for rpc single. [Default: 30s]
+        default_rpc_single_timeout_ms: u32 = 1000 * 30,
```

Comment:
Why such large jumps in timeout defaults?

Reply:
2 seconds is not reasonable for network requests - perhaps 30s is too large, but I feel like it's a good plate to start.